src/Components/Form/Form.js

```diff
@@ -31,6 +52,13 @@ function Form({contacts}) {
     setFormattedContacts(formatted);
   }
 
+  function sendTripData() {
+    console.log(endPoint);
+    console.log(startPoint);
+    openModal();
+    createTrip( {variables: {"startPoint": startPoint, "endPoint": endPoint, "travelMode": travelMode.value}});
```

Comment:
Interesting, I see we can use the startPoint and endPoint props as values, straight up, but we need travelMode.value for that one?